scene/2d/tile_map.cpp

```diff
@@ -1144,6 +1144,24 @@ Array TileMap::get_used_cells() const {
 	return a;
 }
 
+Array TileMap::get_used_cells_by_id(int p_id) const {
+
+	Array a;
+	a.clear();
+	for (Map<PosKey, Cell>::Element *E = tile_map.front(); E; E = E->next()) {
+
+			if (E->value().id == p_id) {
+			Vector2 p(E->key().x, E->key().y);
+			a.push_back(p);
+			
+		}
+		
+	}
+	
+		return a;
```

Comment:
Indentation issue here.

Also for the style, no need to keep empty lines before `}`.